src/wp-includes/script-loader.php

```diff
@@ -3640,3 +3640,19 @@ function _wp_theme_json_webfonts_handler() {
 	add_action( 'wp_enqueue_scripts', $fn_generate_and_enqueue_styles );
 	add_action( 'admin_init', $fn_generate_and_enqueue_editor_styles );
 }
+
+/**
+ * Loads classic theme styles on classic themes.
+ *
+ * This is needed for backwards compatibility for button blocks specifically.
+ */
+function gutenberg_enqueue_classic_theme_styles() {
```

Comment:
No `gutenberg_` in Core pls 😬 Needs either `wp_` or no prefix at all. (I'm not entirely if there's a rule which of those to choose, but to me, dropping the prefix altogether could be okay here.)

```suggestion
function enqueue_classic_theme_styles() {
```

Reply:
Hmm, OTOH, we have some precedent for `wp_enqueue_*`. So let's maybe go with that after all.

```suggestion
function wp_enqueue_classic_theme_styles() {
```